### DISCUSSION TITLE
Home Assistant OS VM

### DISCUSSION THREAD

**Discussion by xtremeideaz** [tteck/Proxmox#123] [General]
Hello, Need your help again. 

trying to migrate my HA installation to a HP Elitedesk 800 G2 mini. its  core i3 3.2ghz  240 SSD 16gb machine running Proxmox.   I used your script to install the HA VM but i get stuck at preparing..... forever. i have deleted the VM over 5 times and still get the same issue when reinstalling. I also tried the manual way without the script. same issue. 

any tips to sort this out? 

Please help

**Comment by tteck** [comment]
Did you enable virtualization and disable secure boot in the bios?

**Reply by xtremeideaz** [reply]
yes i did. 

I checked the Supervisor log and the only red i saw was this

**“Can’t execute restart while a task is in progress”**

**Reply by xtremeideaz** [reply]
Here's a screenshot of the log
![log](https://user-images.githubusercontent.com/79607520/161399493-de93c2a1-0b29-4d9b-bab8-6ba77a94c624.JPG)

**Comment by guarddog13** [comment]
I know this is an old thread but figured I would share this in case someone is having issues with this. 

This actually has been an issue with the Home Assistant OS install image on any version. Here's the solution, if you still have issues after following his directions, then make sure that you also follow post #3 in the same link. 

https://community.home-assistant.io/t/solution-to-a-hanged-preparing-home-installation-screen-during-new-install-on-a-raspberry-pi-4-dns-fix/273803/4

**Reply by nacho-domatix** [reply]
Not sure this is the right thread, but I was wondering why Home Assistant has 3 different installations and none of them are LXC pure container.

**Reply by tteck** [reply]
Home Assistant-Core is Python, no Docker.

**Reply by nacho-domatix** [reply]
That's right. Dumb me, I got wrong when I checked 🤦‍♂️

**Reply by tteck** [reply]
I added it to the repo today 😉

**Reply by nacho-domatix** [reply]
Ahhh, I was just testing it anyway :) 

I'm getting this:
```
Home Assistant-Core should be reachable (after it's initialized) by going to the following URL.
         http://192.168.1.139:8123
root@gnacho:~# pct enter 2022
root@homeassistant:~# cd /srv/homeassistant && python3 -m venv . && source bin/activate && hass
Unable to find configuration. Creating default one in /root/.homeassistant
ERROR: Cannot set filter: libpcap is not available. Cannot compile filter !
2022-09-29 21:22:52.953 ERROR (SyncWorker_0) [scapy.runtime] Cannot set filter: libpcap is not available. Cannot compile filter !
2022-09-29 21:22:53.045 ERROR (SyncWorker_0) [homeassistant.components.dhcp] Cannot watch for dhcp packets without a functional packet filter: libpcap is not available. Cannot compile filter !

```
Anyway is not finished yet and I gotta go to sleep. I'll let you know if I (don't) succeed.

Many thanks again! Awesome work! :)

**Reply by tteck** [reply]
Yeah, those are just logs. Reboot the LXC, and enjoy Dockerless!

**Reply by nacho-domatix** [reply]
I had to Ctrl+C to go back to the terminal. Seems like the script doesn't do it for you. Everything else works fine, I'll create the user and all later :)

Thanks a lot! I love dockerless services (that was the only one actually)

**Reply by tteck** [reply]
When you start HA with `hass` the virtual environment (venv) shows the output until you exit. Kinda like `systemctl status homeassistant`. A tip, use `systemctl restart homeassistant` in the console for a HA reboot.

**Reply by nacho-domatix** [reply]
Ctrl+C did the job. So I'm happy, and everything works perfectly :)
